packages/turf-line-intersect/index.ts

```diff
@@ -19,15 +19,12 @@ import { sweeplineIntersections as findIntersections } from "./lib/sweepline-int
  * @param {GeoJSON} line2 any LineString or Polygon
  * @param {Object} [options={}] Optional parameters
  * @param {boolean} [options.removeDuplicates=true] remove duplicate intersections
- * @param {boolean} [options.ignoreSelfIntersections=false] ignores self-intersections on input features
+ * @param {boolean} [options.ignoreSelfIntersections=true] ignores self-intersections on input features
  * @returns {FeatureCollection<Point>} point(s) that intersect both
  * @example
  * var line1 = turf.lineString([[126, -11], [129, -21]]);
  * var line2 = turf.lineString([[123, -18], [131, -14]]);
  * var intersects = turf.lineIntersect(line1, line2);
- *
```

Comment:
Was the example removed for a reason? If not please add back in so it shows up on turfjs.org.

Reply:
hmm not sure what i did to cause this - i'll revert